src/lib/network/wicked.rb

```diff
@@ -16,5 +16,27 @@ def reload_config(devs)
 
       SCR.Execute(BASH_PATH, "wicked ifreload #{devs.join(" ")}").zero?
     end
+
+    # Reloads configuration for each device named in devs
+    #
+    # @devs [Array] list of device names
```

Comment:
@mvidner I have updated the description with the correct behavior, but the methods just differ in the wicked call, I was thinking in use meta-programming here as it is almost the same code for the 3 methods, but by the moment I decided to use some duplication instead of other code harder to read and test. So suggestions are welcomed :wink: